MoMMI/Modules/chance.py

```diff
@@ -95,3 +95,27 @@
     register_help(__name__, "pick", """Man can you believe this? People actually want to do *fair* 50/50 picks between things? Kids these days.
 
 Fine, just run @MoMMI pick(a,b,c) with as many comma separated values as you need. Normies.""")
+
+# todo
+# support short label codes like qol, bugfix etc, so not search for labels literally
+# filter for emojicracy, just use emoji-modifiers to calc total value of issue, then choose between top ~10ish
+@command("giveissue", r"(?:giveissue)(?:\s+([\w\,]*))?(?:\s+(\w\/))?")
+async def giveissue_command(channel: MChannel, match: Match, message: Message) -> None:
+    channel.send(":hourglass_flowing_sand: Fetching random issue")
+
+    #getting labels
+    t_labels = [x.strip() for x in match.group(1).split(",")] # strip whitespaces
+    params = {"labels" : t_labels.join(",")}
+
+    #getting repo
+    repo = "vgstation-coders/vgstation13"
+    if(match.group(2))
+        repo = match.group(2).strip()
+
+    url = github_url(f"/repos/{repo}/issues", params)
+    issues = await get_github_object(url)
+
+    rand_issue = random.choice(issues).number
```

Comment:
Well you obviously haven't implemented label filtering via the API yet, but I'm kinda worried that this will be too big of a request and you'll have to do pagination.

Reply:
dunno what pagmation is, but if this is a big request, can't i just `await`?

Reply:
also yes, it should already filter labels via the api, but you need to type them out, so "Bug / Fix" or "Easy Fix".
Also labels aren't case-sensitive.

Reply:
Pagination, not pagmation. GitHub only sends 30 items per page when doing these kinds of requests (and you can tell them to up this to 100) which is kind of a nuisance.

Also `github_url` doesn't take a params argument and I have no idea why you'd think it would.

Reply:
meaning i'd have to iterate over the pages? How would i do that.
Also how do i set the limit to 100, i didn't find it in the api docs.

ah shit get_github_object takes em, switched em up.